src/components/ProgressIndicators/StepBar/StepBar.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+import classNames from "classnames/bind";
+
+import { ArrowLink } from "../../Link";
+import styles from "./StepBar.module.scss";
+
+type StepBarProps = {
+  step: number;
+  steps: string[];
+  stepFormatter?: string;
+  onNavigate?: (i: number) => void;
+  className?: string;
+};
+
+export const StepBar: React.FC<StepBarProps> = ({ step, steps, stepFormatter, onNavigate, className }) => {
+  const translate = step >= steps.length ? `${(steps.length - 1) * 100}%` : `${(step - 1) * 100}%`;
+  const width = `${(1 / steps.length) * 100}%`;
```

Comment:
nit: might be cleaner to calculate the number here, and do the string concatenation in place.

Reply:
same, never changed this part